src/main/java/com/urswolfer/gerrit/client/rest/http/GerritRestClient.java

```diff
@@ -252,6 +256,12 @@ private void setRequestBody(String requestBody, HttpRequestBase method) {
         }
     }
 
+    private void setRequestBodyTextPlain(String requestBody, HttpRequestBase method) {
```

Comment:
Probably change this method to a method `setRequestBody` which takes the content type as param? Then it could also be used from the existing `setRequestBody`.

Reply:
yes, agree, will change it